tests/e2e/specs/in-game/reconnecting.spec.js

```diff
@@ -664,3 +662,34 @@ describe('Reconnecting to a game', () => {
     });
   });
 });
+
+describe('Display correct dialog for unavailable game', () => {
+  beforeEach(() => {
+    cy.setupGameAsP0();
+  });
+
+  it('Shows unavailable game dialog, then return home', () => {
+    cy.loadGameFixture(0, {
+      p0Hand: [Card.SEVEN_OF_CLUBS],
+      p0Points: [Card.SEVEN_OF_DIAMONDS, Card.SEVEN_OF_HEARTS],
+      p0FaceCards: [],
+      p1Hand: [],
+      p1Points: [],
+      p1FaceCards: [],
+    });
```

Comment:
Since the test doesn't care which cards are where we can skip loading a flame fixture 
```suggestion
```